package.json

```diff
@@ -1,30 +1,37 @@
 {
-  "name": "backend",
+  "name": "mobilier-backend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "app.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "npm run db && ts-node app.ts",
+    "start:dev": "npm run db && ts-node-dev --respawn --transpile-only app.ts",
+    "db": "docker-compose up -d",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "test": "jest"
```

Comment:
:memo: New scripts:
* db: Will start the DB for local development
* start: Starts the application and ensures that the DB is alive
* start:dev: Starts the application on development mode and ensures that the DB is alive
* lint: Checks files for linting errors
* lint:fix: Checks files for linting errors fixing those than can be automatically fixed by ESLint
* test: Run tests